daisy_workflows/export/export_disk.sh

```diff
@@ -68,8 +68,13 @@ SBOM_UTIL_GCS_ROOT=$(curl -f -H Metadata-Flavor:Google ${URL}/sbom-util-gcs-root
 # Mostly used for windows workflows, set to true if the sbom is already generated and non-empty.
 SBOM_ALREADY_GENERATED=$(curl -f -H Metadata-Flavor:Google ${URL}/sbom-already-generated)
 
-# The sha256 sum for the image tar gz file: used for the sbom
-SHA256_TXT=$(curl -f -H Metadata-Flavor:Google ${URL}/sha256-txt)
+# The sha256 sum local text file path used for the sbom: same functionality as $SBOM_PATH.
+SHA256_PATH=$(curl -f -H Metadata-Flavor:Google ${URL}/sha256-path)
+
+if SBOM_PATH == "" || SHA256_PATH == ""; then
+  echo "ExportFailed: sbom path or sha256 text file path was not passed in"
```

Comment:
can we print both values in the log for easier debugging in case anything goes wrong